Woof/Woof/View/ReusableViews/TextEditorWithPlaceholder.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying TextEditor with a prompt text message for a user.
+struct TextEditorWithPlaceholder: View {
+    /// The text to be shown in the TextEditor.
+    @Binding var message: String
+
+    /// The text that is shown in the textEditor when it’s empty.
+    let placeholder: LocalizedStringKey
```

Comment:
Why do we use the `LocalizedStringKey` here?

Reply:
Here, of course,  we can and should use regular `String`.
We don't need here the key  to look up an entry in a strings file or, for example, strings dictionary file.
My mistake.
I've reread doc about [LocalizedStringKey](https://developer.apple.com/documentation/swiftui/localizedstringkey#).

Fixed it: the `LocalizedStringKey` is replaced by `String`